### PR TITLE
react_people-table-basics

### PR DESCRIPTION
**Pull request by AleksandrYuschenko**
[DEMO LINK](https://aleksandryuschenko.github.io/react_people-table-basics/)

### CODE REVIEW

**Review by Yura-Lysenko — APPROVED** [review]
looks good, well done!)

File: src/components/PersonLink.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person | undefined
+  // eslint-disable-next-line max-len
```
Comment (ivan-kadykalo) [inline src/components/PersonLink.tsx:8]: Why do you need this here?

Comment (ivan-kadykalo) [inline src/components/PersonLink.tsx:8]:
```suggestion
```

**Review by ivan-kadykalo — APPROVED** [review]
GJ🚀

**Comment by AleksandrYuschenko** [comment]
> GJ🚀

**Pull request closed without merging**